src/test/ui/pattern/pat-tuple-underfield.stderr

```diff
@@ -58,27 +58,27 @@ LL |         S(..) => {}
    |           ++
 
 error[E0023]: this pattern has 1 field, but the corresponding tuple variant has 2 fields
-  --> $DIR/pat-tuple-underfield.rs:27:9
+  --> $DIR/pat-tuple-underfield.rs:27:14
    |
 LL |     S(i32, f32),
-   |     ----------- tuple variant defined here
+   |       ---  --- tuple variant has 2 fields
 ...
 LL |         E::S(x) => {}
-   |         ^^^^^^^ expected 2 fields, found 1
+   |         ---- ^ expected 2 fields, found 1
```

Comment:
I'm wondering if we should remove the secondary span for `E::S`. I feel like it is kind of distracting.

Reply:
I'm ok with removing it cases like this. We can check whether the spans are all in the same line to handle cases like

```rust
E::S(
    x,
    y,
) => {}
```
So that in that case we actually point at `E::S` to provide context that otherwise would be missing.

Reply:
r=me after removing the secondary span label with no text in the pattern for cases where there's more than 0 fields, and changing the 0 field case to point at the whole pattern (like it used to).